src/components/Item/VisualizationItem/Item.js

```diff
@@ -69,6 +70,14 @@ export class Item extends Component {
             await pluginManager.fetch(this.props.item)
         )
 
+        try {
+            if (this.props.countInDashboard) {
+                const postFavoriteDataStatistics = await pluginManager.postFavoriteDataStatistics(this.props.item)
```

Comment:
pluginManager has been refactored so that it only contains functions relevant to the maps, event reports, event visualizer.

So instead of adding this function to the pluginManager, just call the api directly.  (and use function `getVisualizationId` from modules/item)

Reply:
Got it. Updated this.